templates/factory.template

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Generated factory for %class%
+ */
+
+%namespace%
```

Comment:
When substitutions are made, is a newline added after the namespace? If not, it should.

Reply:
Yes it is, there are test cases that actually cover this. Same as for the use statements